RELEASE-NOTES.txt

```diff
@@ -3,6 +3,7 @@
  
 14.9
 -----
+* [**] Block editor: Avoid crash when editor gets into invalid state
```

Comment:
Since we already sent the translations for `14.9` and considering this may not be a crucial update, I'd consider removing it from the release notes or marking it as internal. We have had some issues in the past where some translations were marked as fuzzy because they were altered after submission and didn't get translated. I don't think it'll be a big issue when we are adding a completely new bullet point, but to be on the safe side, I'd suggest removing it.

We also don't have any room left in the shorter or longer versions of the release notes for this version, so we'd have to remove something else for it.

Reply:
Thanks @oguzkocer ! Does the fact that this is fixing a [really prevalent crash](https://github.com/wordpress-mobile/WordPress-Android/issues/9832#issuecomment-630952570) (140k crashes for 20k users in the last 90 days) change your assessment of how we should handle this release note at all? I don't feel strongly.

Reply:
It could - in some cases. From a user's perspective, even if I saw this release note, it'd not mean that it's a big deal to me. (personal opinion) In that regard, I don't feel our users would miss a crucial information. Having said that, I only brought it up as a suggestion, if you want me to see if editorial can squeeze it in, I am happy to ask them. (they'll most likely ask some details btw)

Reply:
> even if I saw this release note, it'd not mean that it's a big deal to me

That's a good point since it's not like there is a clear path to this crash that I'm able to call out in the release note. I'll go ahead and take this release note out. 👍